### PR TITLE
Cleanup: delete row-wise loading code

### PR DESCRIPTION
**Pull request by sduzh**
Related issue #1012

Delete the receiver part code of row-wise loading.

### CODE REVIEW

**Comment by stdpain** [comment]
run starrocks_be_unittest

**Review by chaoyli — DISMISSED** [review]

File: be/src/runtime/tablets_channel.cpp [hunk 1]
```diff
@@ -347,79 +269,42 @@ Status TabletsChannel::reduce_mem_usage_async(const std::set<int64_t>& flush_tab
...
-        return vectorized_writer->flush_memtable_async();
+    if (vectorized_writer == nullptr || max_consume == 0) {
+        // barely not happend, just return OK
+        return Status::OK();
     }
+    VLOG(3) << "pick the delta writer to flush, with mem consumption: " << max_consume << ", channel key: " << _key;
+    *tablet_mem_consumption = max_consume;
+    std::lock_guard<std::mutex> l(_tablet_locks[*tablet_id & k_shard_size]);
+    return vectorized_writer->flush_memtable_async();
     return Status::OK();
```
Comment (wyb) [inline be/src/runtime/tablets_channel.cpp:290]: remove the last return.

File: be/src/runtime/tablets_channel.cpp [hunk 2]
```diff
@@ -347,79 +269,42 @@ Status TabletsChannel::reduce_mem_usage_async(const std::set<int64_t>& flush_tab
...
+        vectorized_writer = it->second;
     }
 
-    if (_is_vectorized) {
-        std::lock_guard<std::mutex> l(_tablet_locks[tablet_id & k_shard_size]);
-        return vectorized_writer->wait_memtable_flushed();
-    }
+    std::lock_guard<std::mutex> l(_tablet_locks[tablet_id & k_shard_size]);
+    return vectorized_writer->wait_memtable_flushed();
     return Status::OK();
```
Comment (wyb) [inline be/src/runtime/tablets_channel.cpp:308]: remove the last return.

Comment (sduzh) [reply be/src/runtime/tablets_channel.cpp]: updated

File: be/src/runtime/tablets_channel.cpp (hunk 1 above)
Comment (sduzh) [reply be/src/runtime/tablets_channel.cpp]: updated

**Review by wyb — DISMISSED** [review]

**Review by chaoyli — APPROVED** [review]

**Review by Seaven — APPROVED** [review]

**Pull request merged**